src/cli/serve/serve.js

```diff
@@ -137,16 +132,7 @@ function applyConfigOverrides(rawConfig, opts, extraCliOptions) {
   if (opts.logFile) set('logging.dest', opts.logFile);
 
   set('plugins.scanDirs', _.compact([].concat(get('plugins.scanDirs'), opts.pluginDir)));
-  set(
-    'plugins.paths',
-    _.compact(
-      [].concat(
-        get('plugins.paths'),
-        opts.pluginPath,
-        XPACK_INSTALLED && !opts.oss ? [XPACK_DIR] : []
-      )
-    )
-  );
+  set('plugins.paths', _.compact([].concat(get('plugins.paths'), opts.pluginPath)));
```

Comment:
This was only used to specify the legacy xpack plugin's path. KP's xpack path is handled in `packages/kbn-config/src/env.ts`